agr_literature_service/api/models/mod_taxon_model.py

```diff
@@ -0,0 +1,37 @@
+"""
+mod_taxon_model.py
+==================
+"""
+# from datetime import datetime
+# import pytz
+
+from typing import Dict
+# from sqlalchemy import (Column, Integer, String, ForeignKey, DateTime)
+from sqlalchemy import (Column, Integer, ForeignKey)
+from agr_literature_service.api.database.base import Base
+from agr_literature_service.api.models.audited_model import AuditedModel
+
+
+class ModTaxonModel(AuditedModel, Base):
+    __tablename__ = "mod_taxon"
+    __versioned__: Dict = {}
+
+    mod_taxon_id = Column(
+        Integer,
+        primary_key=True,
+        autoincrement=True
+    )
+
+    mod_id = Column(
+        Integer,
+        ForeignKey("mod.mod_id"),
+        unique=False,
+        nullable=False
+    )
+
+    # taxon_id
+    taxon = Column(
```

Comment:
From the conversation with Sierra and Olin
Sierra : `so your column name could be "taxon_id" (that could also be the slot name for the class).
in our alliance model, we have the slot "taxon" which fulfills this purpose already`
and
Olin : `yes we store the full curie in the curie field and when linking typically it would be taxon_id which would point to the curie field`
I think I was confused by her mentioning the slot "taxon" but the column name being "taxon_id".  I guess they're saying the column should be taxon_id.  Up to you whether we should rename the column, so far we're using _id to mean an internal database ID, right ?
They both seem to agree that it should store the curie String() though

Reply:
It is now taxon. We can change it later if it becomes important.

Reply:
Sounds good